posts/2024/2024_05_27_Inigo_week_0.rst

```diff
@@ -0,0 +1,25 @@
+Community Bonding Period Summary
+===================================================
+
+.. post:: May 27 2024
+    :author: Iñigo Tellaetxe
+    :tags: google
+    :category: gsoc
+
+
+
+What I did this week
+~~~~~~~~~~~~~~~~~~~~
+I talked with my mentors and we agreed on the first tasks to carry out. Jon Haitz provided me with the data he used to train his AutoEncoder (AE) network. These are 2 datasets: Tractoinferno and FiberCup.
```

Comment:
I did not provide TractoInferno, right?

Reply:
I also thought so, but if you check the [Zenodo entry for FINTA/CINTA/GESTA/FIESTA](https://zenodo.org/records/7562790) there is a file called `data_tractoinferno_hcp_qbx.hdf5`, so I guess it does come from TractoInferno. Am I wrong?

Reply:
OK, you're right. I was more meaning that for FiberCup I had to dig into my archives, but no need to add more detail. Resolving.